tape/src/main/java/com/squareup/tape2/QueueFile.java

```diff
@@ -761,4 +761,9 @@ public QueueFile build() throws IOException {
       }
     }
   }
+
+  @SuppressWarnings({"unchecked", "TypeParameterUnusedInFormals"})
+  static <T extends Throwable> T getSneakyThrowable(Throwable t) throws T {
```

Comment:
Should we add a note for future viewers as to why we throwing sneakily here?

Reply:
Good idea. Will update tonight.